dbt_project.yml

```diff
@@ -2,7 +2,7 @@ name: 'snowplow_web'
 version: '0.4.0'
 config-version: 2
 
-require-dbt-version: [">=0.18.0", "<0.20.0"]
+require-dbt-version: [">=0.20.0", "<0.22.0"]
```

Comment:
v1 is out now. Should we remove the upper bound requirement? Or at least go `>=0.20.0` and `<2.0.0`?

Reply:
Potentially. Our package leverages some of the [artifacts](https://docs.getdbt.com/reference/artifacts/dbt-artifacts#when-are-artifacts-produced) generated by dbt and from what I understand the schema of these can change in minor releases going forward. As a result I think there is still potential for breaking changes even with minor dbt releases. How do you think we best handle this?

Reply:
Oh golly gosh. I'd really want to avoid us having to make a release every time dbt does just so we stay compatible.

I'm tempted to suggest we go for `>=0.20.0` and `<2.0.0` and only produce a fix if dbt makes a "breaking change" in a minor release, which sees us raising the lower version limit to that breaking version(?). I appreciate our package will break in that circumstance but I think thats better than us constantly being behind the latest release?

Reply:
Sounds fair.  Damned if we do damned if we don't